src/core/api.h

```diff
@@ -253,6 +253,22 @@ MsQuicGetParam(
         void* Buffer
     );
 
+_IRQL_requires_max_(PASSIVE_LEVEL)
+void
+QUIC_API
+MsQuicHandleAddRef(
```

Comment:
I'd prefer not to have `Handle` in the title. None of the other generic functions have it (`GetParam`, `SetParam`).

Reply:
Other option is I change the other one to MsQuicLibraryAddRef

Reply:
Ah, I see. Ok. I'm fine with this then.